### PR TITLE
[trafficserver] update

### PR DESCRIPTION
**Pull request by 0x34d**
For PR: https://github.com/apache/trafficserver/pull/10151

DON'T merge this.

### CODE REVIEW

**Comment by Alan32Liu** [comment]
Converted to a draft since it is not ready to merge.

File: projects/trafficserver/Dockerfile [hunk 1]
```diff
@@ -14,9 +14,7 @@
 #
 ################################################################################
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y autoconf automake libtool pkg-config libpcre3-dev zlib1g-dev
-RUN git clone --depth 1 https://github.com/apache/trafficserver
-RUN git clone https://github.com/0x34d/oss-fuzz-bloat
+RUN apt-get update && apt-get install -y libpcre3-dev zlib1g-dev cmake libssl-dev
+RUN git clone --depth 1 -b master https://github.com/0x34d/trafficserver.git
```
[the remark this answers is not in the document]
Comment (0x34d) [reply projects/trafficserver/Dockerfile:18]: Yes.

**Comment by Alan32Liu** [comment]
@0x34d Is this ready for review?
Thanks: )

**Comment by 0x34d** [comment]
> @0x34d Is this ready for review?

Yes.

**Review by Alan32Liu — APPROVED** [review]

**Pull request merged**